### DISCUSSION TITLE
How to request adding a new language 'urdu'

### DISCUSSION THREAD

**Discussion by lalamax3d** [suno-ai/bark#63] [General]
i see, its not in languages list. can i be of some help in collecting / labeling / providing data.
any psudo steps / guidelines...if individual can add it by himself.

**Comment by gkucsko** [comment]
thanks! please add to a unified tread here: https://github.com/suno-ai/bark/discussions/111

**Comment by Ratha-Chhom** [comment]
This is cool. But please kindly add Khmer Language (Cambodian Language).

**Comment by Nirmalraj10567** [comment]
Add Tamil language worldest old language
